pyfakefs/fake_filesystem_unittest.py

```diff
@@ -134,6 +139,12 @@ class Patcher(object):
     # it appears that adding  'py', 'pytest', '_pytest' to SKIPNAMES will help
     SKIPNAMES = set(['os', 'path', 'tempfile', 'io'])
 
+    # To avoid stubbing out path because another top=level module named path is present
```

Comment:
I incorporated this excellent explanation into the `patch_path` docstring above.